gui/launcher.cpp

```diff
@@ -282,13 +286,26 @@ void LauncherDialog::updateListing() {
 
 			while (pos < size && (scumm_stricmp(description.c_str(), l[pos].c_str()) > 0))
 				pos++;
+
+			EngineMan.findGame(gameid, &plugin);
```

Comment:
When using dynamic plugins, this causes the engine plugins to be repeatedly loaded and unloaded. Is that going to be too slow?